### ISSUE TITLE
Exception: Incorrect parameter. (Exception from HRESULT: 0x80070057 (E_INVALIDARG))

### ISSUE THREAD

**Issue by gigios** [sboulema/TSVN#52]
I'm using the extension for the first time in a my solution, with the new Visual Studio 2019.
I have some solution folders where there are some document files (release_notes.txt & c).
Seems that TSVN is not able to manage these files, every time, for every command, raise the exception (exception translated from the Italian language, I'm not sure that the exception description is exactly that):
Incorrect parameter. (Exception from HRESULT: 0x80070057 (E_INVALIDARG))

All these 'solution folder' are present also on disk inside the working copy root!

**Comment by sboulema** [comment]
Did you install TortoiseSVN with the client tools?
https://github.com/sboulema/TSVN/wiki/Menu-items-not-working#tortoisesvn

**Comment by gigios** [comment]
Yes, I have checked today again the installation and the client tools are installed!
In the normal files, stored inside a project, the TSVN extension works fine, the problem seems only for the files, inside the solution folder (that, I repeat, are not real folders, probably this is the problem).

**Comment by triathlonchris** [comment]
I find out that I get this Exception only if I have the Manage NuGet Packages window open. Pershaps this helps. If I close the window all works fine again.

**Comment by ehsanelahimalik** [comment]
In my case it was the "Publish" Website window.
Works fine when I close it, when the window is open Commit or update do not work.
